src/test/java/vg/civcraft/mc/civmodcore/serialization/Tester.java

```diff
@@ -0,0 +1,32 @@
+package vg.civcraft.mc.civmodcore.serialization;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import vg.civcraft.mc.civmodcore.api.NBTCompound;
+
+public class Tester {
+
+	@Test
+	public void testSerialization() {
```

Comment:
nice

Reply:
Thanks :) it does mean another dependency (junit) but I hope that's okay considering its purpose